src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/FindBreakpointEvidenceSpark.java

```diff
@@ -299,6 +327,30 @@ private static EvidenceScanResults getMappedQNamesSet(
         return new EvidenceScanResults(readMetadata, intervals, intervalsAndEvidenceTargetLinks._2(), qNamesMultiMap);
     }
 
+    static SVIntervalTree<SVInterval> findGenomewideHighCoverageIntervalsToIgnore(final FindBreakpointEvidenceSparkArgumentCollection params,
+                                                                                  final ReadMetadata readMetadata,
+                                                                                  final JavaSparkContext ctx,
+                                                                                  final SAMFileHeader header,
+                                                                                  final JavaRDD<GATKRead> unfilteredReads,
+                                                                                  final SVReadFilter filter,
+                                                                                  final Logger logger,
+                                                                                  final Broadcast<ReadMetadata> broadcastMetadata) {
+        final long nRefBases = broadcastMetadata.getValue().getNRefBases();
+        final List<SVInterval> depthIntervals = new ArrayList<>((int) ((float) nRefBases / DEPTH_WINDOW_SIZE));
+        for (final SAMSequenceRecord sequenceRecord : header.getSequenceDictionary().getSequences()) {
+            for (int i = 1; i < sequenceRecord.getSequenceLength(); i = i + DEPTH_WINDOW_SIZE) {
+                depthIntervals.add(new SVInterval(readMetadata.getContigID(sequenceRecord.getSequenceName()), i, Math.min(sequenceRecord.getSequenceLength(), i + DEPTH_WINDOW_SIZE)));
```

Comment:
readMetadata.getContigID(sequenceRecord.getSequenceName()) is a loop invariant (and involves a hashed lookup).  Pull it out as a temporary variable.
(So is sequenceRecord.getSequenceLength(), but that's fast to compute, so it doesn't matter.)

Reply:
done